<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_new_task</class>
 <widget class="QWidget" name="frm_new_task">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Neue Aufgabe erstellen</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QWidget" name="wg_main_new_task" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>550</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="lb_new_task">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Neue Aufgabe erstellen</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="wg_title" native="true">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <property name="horizontalSpacing">
          <number>33</number>
         </property>
         <property name="verticalSpacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="lb_title">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="le_title"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="wg_category" native="true">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="QLabel" name="lb_category">
           <property name="text">
            <string>Kategorie:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_category">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>Arbeit</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Zu Hause</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="wg_status" native="true">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>33</number>
         </property>
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="QLabel" name="lb_status">
           <property name="text">
            <string>Status:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_status">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>in Bearbeitung</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>noch nicht angefangen</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Erledigt</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="fr_description">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="lb_description">
           <property name="text">
            <string>Beschreibung der Aufgabe:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="te_description">
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFontComboBox" name="fcb_description"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="wg_time" native="true">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QCheckBox" name="cbx_show_timeedit">
           <property name="text">
            <string>Termin festlegen</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="wg_time_expand" native="true">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QDateEdit" name="ded_description">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
           <property name="date">
            <date>
             <year>2025</year>
             <month>6</month>
             <day>19</day>
            </date>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTimeEdit" name="ted_description">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="time">
            <time>
             <hour>10</hour>
             <minute>20</minute>
             <second>0</second>
            </time>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="wg_new_task_buttons" native="true">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pb_close">
           <property name="minimumSize">
            <size>
             <width>110</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>140</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>Beenden</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pb_finish_task">
           <property name="minimumSize">
            <size>
             <width>110</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>140</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>Abschließen</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pb_create_task">
           <property name="minimumSize">
            <size>
             <width>110</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>140</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>Erstellen</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pb_close</sender>
   <signal>clicked()</signal>
   <receiver>frm_new_task</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>686</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbx_show_timeedit</sender>
   <signal>clicked(bool)</signal>
   <receiver>wg_time_expand</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>581</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>632</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
